docs/wiki/ModulesDesign.md

```diff
@@ -537,3 +538,45 @@ Parameter files in CARML leverage the common `deploymentParameters.json` schema
 - Likewise, the `name` parameter we have in most modules should give some indication of the file it was deployed with. For example, a `min.parameters.json` parameter file for the virtual network module may have a `name` property with the value `sxx-az-vnet-min-001` where `min` relates to the prefix of the parameter file itself.
 - A module should have as many parameter files as it needs to evaluate all parts of the module's functionality.
 - Sensitive data should not be stored inside the parameter file but rather be injected by the use of [tokens](./ParameterFileTokens.md) or via a [key vault reference](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/key-vault-parameter?tabs=azure-cli#reference-secrets-with-static-id).
+
+# Resource Removal
+
+Removal step is triggered after the deployment completes. This is used for several reasons:
+- Make sure to keep the validation subscription cost as low as possible.
+- Enable testing of the full module deployment at every run.
+
+The default removal procedure works fine for most of the modules created so far, so it's likely you won't have to change anything to make your module to be removed correctly after deployment.
```

Comment:
```suggestion
The default removal procedure works fine for most of the modules created so far, so it's likely you won't have to change anything to make a new module to be removed correctly after deployment.
```

Reply:
I will change it to "the module you're editing"